lib/utils/mixin.html

```diff
@@ -38,6 +38,7 @@
    * @memberof Polymer
    * @template T
    * @param {T} mixin ES6 class expression mixin to wrap
+   * @return {T}
```

Comment:
From what I remember, the magic voodoo that Closure Compiler uses for mixins doesn't like a return type here. Can you double check that `gulp lint-closure` doesn't produce any warnings?

Reply:
> Generally LGTM, minus that nit and need to double-check closure with return type on deduping mixin.

Added some items to the CHANGELOG. Where should I mention the Node version in the README? It doesn't currently mention the gulp script. (async/await was actually 7.6 btw).

Reply:
> From what I remember, the magic voodoo that Closure Compiler uses for mixins doesn't like a return type here. Can you double check that gulp lint-closure doesn't produce any warnings?

I ran `gulp lint`, and it produces a lot of warnings. Nothing about `dedupingMixin` that I can see, though. Here's the log, does it look normal?: https://gist.github.com/aomarks/3adb586df000a36e682cc4f87137606b

Reply:
Closure is compiling cleanly now.